src/main/scala/is/hail/utils/Py4jUtils.scala

```diff
@@ -25,13 +25,12 @@ trait Py4jUtils {
     list
   }
 
-  def parseIntervalList(strs: java.util.ArrayList[String]): IntervalTree[Locus] =
-    IntervalTree(Locus.parseIntervals(strs.asScala.toArray), prune = true)
+  def parseIntervalList(strs: java.util.ArrayList[String]): IntervalTree[Locus, Unit] =
+    IntervalTree(Locus.parseIntervals(strs.asScala.toArray), noisy = true)
 
-  def makeIntervalList(intervals: java.util.ArrayList[Interval[Locus]]): IntervalTree[Locus] =
-    IntervalTree(intervals.asScala.toArray, prune = true)
+  def makeIntervalList(intervals: java.util.ArrayList[Interval[Locus]]): IntervalTree[Locus, Unit] =
+    IntervalTree(intervals.asScala.toArray, noisy = true)
 
-  // FIXME: don't use vector
   def arrayListToISeq[T](al: java.util.ArrayList[T]): IndexedSeq[T] = al.asScala.toIndexedSeq
```

Comment:
the `new IndexedSeq[T] { ... } ` turns out not to be serializable -- this has caused problems in the past. This thing is pretty much only used for function arguments. Will change the comment.